kernel_gateway/tests/services/cell/test_parser.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from kernel_gateway.services.cell.parser import *
+import unittest
+import re
+class TestAPICellParserUtils(unittest.TestCase):
```

Comment:
We never had tests for this code before? The PR reviewer is slipping ... :sleeping: